### PR TITLE
add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by kuangcheng66**
1. bug-report.md
2. enhancement.md
3. flaking-test.md
4. good-first.md
5. question.md

### CODE REVIEW

**Comment by kuangcheng66** [comment]
@duanmengkk

File: .github/ISSUE_TEMPLATE/question.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Question
+about: Question relating to Kosmos.
+labels: kind/question
+
+---
+
+<!-- Please use this template while asking a new question. Thanks!
+Please make sure you have read the FAQ and searched the issue list.
+- FAQ: https://github.com/Kosmos-io/Kosmos/blob/master/docs/frequently-asked-questions.md
+- Issues: https://github.com/Kosmos-io/Kosmos/issues?q=is%3Aissue+
```
Comment (duanmengkk) [inline .github/ISSUE_TEMPLATE/question.md]: https://github.com/kosmos-io/kosmos/issues

**Comment by wuyingjun-lucky** [comment]
@kuangcheng66  Should we be going on this pr？

File: .github/ISSUE_TEMPLATE/question.md (hunk 1 above)
Comment (kuangcheng66) [reply .github/ISSUE_TEMPLATE/question.md]: ok. I will update it as soon as possible

**Comment by duanmengkk** [comment]
you can use rebase to squash the commit

**Comment by wuyingjun-lucky** [comment]
@kuangcheng66  you can rebase the commits to avoid the upstream merge commits

**Comment by kuangcheng66** [comment]
ok

**Comment by kuangcheng66** [comment]
@wuyingjun-lucky @duanmengkk

**Pull request merged**